jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/editor/CodeWhispererEnterHandler.kt

```diff
@@ -12,18 +13,13 @@ import software.aws.toolkits.telemetry.CodewhispererAutomatedTriggerType
 import software.aws.toolkits.telemetry.CodewhispererTriggerType
 
 class CodeWhispererEnterHandler(private val originalHandler: EditorActionHandler) :
```

Comment:
```suggestion
class CodeWhispererEnterHandler(originalHandler: EditorActionHandler) :
```

i think we can drop the val now?

Reply:
yes -- it's already a field in EnterHandler